ner_v1/api.py

```diff
@@ -290,8 +294,14 @@ def phone_number(request):
 
     """
     try:
-        parameters_dict = get_parameters_dictionary(request)
-        ner_logger.debug('Start: %s ' % parameters_dict[PARAMETER_ENTITY_NAME])
+
+        parameters_dict = {}
+        if request.method == "POST":
+            parameters_dict = parse_post_request(request)
+            ner_logger.debug('Start POST %s ' % parameters_dict[PARAMETER_ENTITY_NAME])
+        elif request.method == "GET":
+            parameters_dict = get_parameters_dictionary(request)
+            ner_logger.debug('Start: GET %s ' % parameters_dict[PARAMETER_ENTITY_NAME])
```

Comment:
This code block is repeated a lot of times, why not make it a function ?

Reply:
Agree. Pushed the changes